src/contract/userop/index.ts

```diff
@@ -0,0 +1,86 @@
+import { UserOperationEventEvent } from "userop/dist/typechain/EntryPoint";
+import { VerifiedWallet } from "../../wallet";
+import { ethers, utils } from "ethers";
+import { Client, Presets } from "userop";
+
+interface Params {
+    apiKey:string, 
+    paymasterUrl:string,
+    wallet:VerifiedWallet
+    signer:VerifiedWallet
+    value:string
+}
+
+const response: { transactionHash: string | null, result: UserOperationEventEvent | null } = { transactionHash: null, result: null }
+export type UseropResponse = typeof response;
+
+ class VerifiedContract {
+
+    constructor(){
```

Comment:
@herberthk 

I told you that we should not change the constructor or other functions of the VerifiedContract, otherwise it will break the SDK contracts. The constructor which we have is 

```
constructor(address: string, abi: string, signer: VerifiedWallet | Signer) {
        this.signer = signer;
        this.abiInterface = new utils.Interface(abi)
        this.contract = new ethers.Contract(address, this.abiInterface, signer);
    }
```